package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-step-functions",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
Could you turn back to 1.5.0?
This should be updated while releasing a new version.